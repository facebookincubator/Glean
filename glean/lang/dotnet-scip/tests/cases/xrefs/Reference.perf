{ "@generated": null, "scip.Reference.1": 82 }