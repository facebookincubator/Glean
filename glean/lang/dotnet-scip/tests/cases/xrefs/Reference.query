query: scip.Reference _
perf: True
transform: [gensort,[]]