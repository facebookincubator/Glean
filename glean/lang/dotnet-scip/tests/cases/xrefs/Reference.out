[
  "@generated",
  {
    "key": {
      "symbol": { "key": "scip-dotnet nuget . . Collections/" },
      "location": {
        "key": {
          "file": { "key": "obj/Debug/net7.0/src.GlobalUsings.g.cs" },
          "range": {
            "lineBegin": 3,
            "columnBegin": 29,
            "lineEnd": 3,
            "columnEnd": 39
          }
        }
      }
    }
  },
  {
    "key": {
      "symbol": { "key": "scip-dotnet nuget . . Generic/" },
      "location": {
        "key": {
          "file": { "key": "obj/Debug/net7.0/src.GlobalUsings.g.cs" },
          "range": {
            "lineBegin": 3,
            "columnBegin": 41,
            "lineEnd": 3,
            "columnEnd": 47
          }
        }
      }
    }
  },
  {
    "key": {
      "symbol": { "key": "scip-dotnet nuget . . Http/" },
      "location": {
        "key": {
          "file": { "key": "obj/Debug/net7.0/src.GlobalUsings.g.cs" },
          "range": {
            "lineBegin": 6,
            "columnBegin": 33,
            "lineEnd": 6,
            "columnEnd": 36
          }
        }
      }
    }
  },
  {
    "key": {
      "symbol": { "key": "scip-dotnet nuget . . IO/" },
      "location": {
        "key": {
          "file": { "key": "obj/Debug/net7.0/src.GlobalUsings.g.cs" },
          "range": {
            "lineBegin": 4,
            "columnBegin": 29,
            "lineEnd": 4,
            "columnEnd": 30
          }
        }
      }
    }
  },
  {
    "key": {
      "symbol": { "key": "scip-dotnet nuget . . Linq/" },
      "location": {
        "key": {
          "file": { "key": "obj/Debug/net7.0/src.GlobalUsings.g.cs" },
          "range": {
            "lineBegin": 5,
            "columnBegin": 29,
            "lineEnd": 5,
            "columnEnd": 32
          }
        }
      }
    }
  },
  {
    "key": {
      "symbol": { "key": "scip-dotnet nuget . . Net/" },
      "location": {
        "key": {
          "file": { "key": "obj/Debug/net7.0/src.GlobalUsings.g.cs" },
          "range": {
            "lineBegin": 6,
            "columnBegin": 29,
            "lineEnd": 6,
            "columnEnd": 31
          }
        }
      }
    }
  },
  {
    "key": {
      "symbol": { "key": "scip-dotnet nuget . . Reflection/" },
      "location": {
        "key": {
          "file": {
            "key": "obj/Debug/net7.0/.NETCoreApp,Version=v7.0.AssemblyAttributes.cs"
          },
          "range": {
            "lineBegin": 3,
            "columnBegin": 14,
            "lineEnd": 3,
            "columnEnd": 23
          }
        }
      }
    }
  },
  {
    "key": {
      "symbol": { "key": "scip-dotnet nuget . . Reflection/" },
      "location": {
        "key": {
          "file": { "key": "obj/Debug/net7.0/src.AssemblyInfo.cs" },
          "range": {
            "lineBegin": 11,
            "columnBegin": 14,
            "lineEnd": 11,
            "columnEnd": 23
          }
        }
      }
    }
  },
  {
    "key": {
      "symbol": { "key": "scip-dotnet nuget . . Reflection/" },
      "location": {
        "key": {
          "file": { "key": "obj/Debug/net7.0/src.AssemblyInfo.cs" },
          "range": {
            "lineBegin": 13,
            "columnBegin": 19,
            "lineEnd": 13,
            "columnEnd": 28
          }
        }
      }
    }
  },
  {
    "key": {
      "symbol": { "key": "scip-dotnet nuget . . Reflection/" },
      "location": {
        "key": {
          "file": { "key": "obj/Debug/net7.0/src.AssemblyInfo.cs" },
          "range": {
            "lineBegin": 14,
            "columnBegin": 19,
            "lineEnd": 14,
            "columnEnd": 28
          }
        }
      }
    }
  },
  {
    "key": {
      "symbol": { "key": "scip-dotnet nuget . . Reflection/" },
      "location": {
        "key": {
          "file": { "key": "obj/Debug/net7.0/src.AssemblyInfo.cs" },
          "range": {
            "lineBegin": 15,
            "columnBegin": 19,
            "lineEnd": 15,
            "columnEnd": 28
          }
        }
      }
    }
  },
  {
    "key": {
      "symbol": { "key": "scip-dotnet nuget . . Reflection/" },
      "location": {
        "key": {
          "file": { "key": "obj/Debug/net7.0/src.AssemblyInfo.cs" },
          "range": {
            "lineBegin": 16,
            "columnBegin": 19,
            "lineEnd": 16,
            "columnEnd": 28
          }
        }
      }
    }
  },
  {
    "key": {
      "symbol": { "key": "scip-dotnet nuget . . Reflection/" },
      "location": {
        "key": {
          "file": { "key": "obj/Debug/net7.0/src.AssemblyInfo.cs" },
          "range": {
            "lineBegin": 17,
            "columnBegin": 19,
            "lineEnd": 17,
            "columnEnd": 28
          }
        }
      }
    }
  },
  {
    "key": {
      "symbol": { "key": "scip-dotnet nuget . . Reflection/" },
      "location": {
        "key": {
          "file": { "key": "obj/Debug/net7.0/src.AssemblyInfo.cs" },
          "range": {
            "lineBegin": 18,
            "columnBegin": 19,
            "lineEnd": 18,
            "columnEnd": 28
          }
        }
      }
    }
  },
  {
    "key": {
      "symbol": { "key": "scip-dotnet nuget . . Reflection/" },
      "location": {
        "key": {
          "file": { "key": "obj/Debug/net7.0/src.AssemblyInfo.cs" },
          "range": {
            "lineBegin": 19,
            "columnBegin": 19,
            "lineEnd": 19,
            "columnEnd": 28
          }
        }
      }
    }
  },
  {
    "key": {
      "symbol": { "key": "scip-dotnet nuget . . Runtime/" },
      "location": {
        "key": {
          "file": {
            "key": "obj/Debug/net7.0/.NETCoreApp,Version=v7.0.AssemblyAttributes.cs"
          },
          "range": {
            "lineBegin": 4,
            "columnBegin": 27,
            "lineEnd": 4,
            "columnEnd": 33
          }
        }
      }
    }
  },
  {
    "key": {
      "symbol": { "key": "scip-dotnet nuget . . System/" },
      "location": {
        "key": {
          "file": { "key": "Program.cs" },
          "range": {
            "lineBegin": 9,
            "columnBegin": 7,
            "lineEnd": 9,
            "columnEnd": 12
          }
        }
      }
    }
  },
  {
    "key": {
      "symbol": { "key": "scip-dotnet nuget . . System/" },
      "location": {
        "key": {
          "file": {
            "key": "obj/Debug/net7.0/.NETCoreApp,Version=v7.0.AssemblyAttributes.cs"
          },
          "range": {
            "lineBegin": 2,
            "columnBegin": 7,
            "lineEnd": 2,
            "columnEnd": 12
          }
        }
      }
    }
  },
  {
    "key": {
      "symbol": { "key": "scip-dotnet nuget . . System/" },
      "location": {
        "key": {
          "file": {
            "key": "obj/Debug/net7.0/.NETCoreApp,Version=v7.0.AssemblyAttributes.cs"
          },
          "range": {
            "lineBegin": 3,
            "columnBegin": 7,
            "lineEnd": 3,
            "columnEnd": 12
          }
        }
      }
    }
  },
  {
    "key": {
      "symbol": { "key": "scip-dotnet nuget . . System/" },
      "location": {
        "key": {
          "file": {
            "key": "obj/Debug/net7.0/.NETCoreApp,Version=v7.0.AssemblyAttributes.cs"
          },
          "range": {
            "lineBegin": 4,
            "columnBegin": 20,
            "lineEnd": 4,
            "columnEnd": 25
          }
        }
      }
    }
  },
  {
    "key": {
      "symbol": { "key": "scip-dotnet nuget . . System/" },
      "location": {
        "key": {
          "file": { "key": "obj/Debug/net7.0/src.AssemblyInfo.cs" },
          "range": {
            "lineBegin": 10,
            "columnBegin": 7,
            "lineEnd": 10,
            "columnEnd": 12
          }
        }
      }
    }
  },
  {
    "key": {
      "symbol": { "key": "scip-dotnet nuget . . System/" },
      "location": {
        "key": {
          "file": { "key": "obj/Debug/net7.0/src.AssemblyInfo.cs" },
          "range": {
            "lineBegin": 11,
            "columnBegin": 7,
            "lineEnd": 11,
            "columnEnd": 12
          }
        }
      }
    }
  },
  {
    "key": {
      "symbol": { "key": "scip-dotnet nuget . . System/" },
      "location": {
        "key": {
          "file": { "key": "obj/Debug/net7.0/src.AssemblyInfo.cs" },
          "range": {
            "lineBegin": 13,
            "columnBegin": 12,
            "lineEnd": 13,
            "columnEnd": 17
          }
        }
      }
    }
  },
  {
    "key": {
      "symbol": { "key": "scip-dotnet nuget . . System/" },
      "location": {
        "key": {
          "file": { "key": "obj/Debug/net7.0/src.AssemblyInfo.cs" },
          "range": {
            "lineBegin": 14,
            "columnBegin": 12,
            "lineEnd": 14,
            "columnEnd": 17
          }
        }
      }
    }
  },
  {
    "key": {
      "symbol": { "key": "scip-dotnet nuget . . System/" },
      "location": {
        "key": {
          "file": { "key": "obj/Debug/net7.0/src.AssemblyInfo.cs" },
          "range": {
            "lineBegin": 15,
            "columnBegin": 12,
            "lineEnd": 15,
            "columnEnd": 17
          }
        }
      }
    }
  },
  {
    "key": {
      "symbol": { "key": "scip-dotnet nuget . . System/" },
      "location": {
        "key": {
          "file": { "key": "obj/Debug/net7.0/src.AssemblyInfo.cs" },
          "range": {
            "lineBegin": 16,
            "columnBegin": 12,
            "lineEnd": 16,
            "columnEnd": 17
          }
        }
      }
    }
  },
  {
    "key": {
      "symbol": { "key": "scip-dotnet nuget . . System/" },
      "location": {
        "key": {
          "file": { "key": "obj/Debug/net7.0/src.AssemblyInfo.cs" },
          "range": {
            "lineBegin": 17,
            "columnBegin": 12,
            "lineEnd": 17,
            "columnEnd": 17
          }
        }
      }
    }
  },
  {
    "key": {
      "symbol": { "key": "scip-dotnet nuget . . System/" },
      "location": {
        "key": {
          "file": { "key": "obj/Debug/net7.0/src.AssemblyInfo.cs" },
          "range": {
            "lineBegin": 18,
            "columnBegin": 12,
            "lineEnd": 18,
            "columnEnd": 17
          }
        }
      }
    }
  },
  {
    "key": {
      "symbol": { "key": "scip-dotnet nuget . . System/" },
      "location": {
        "key": {
          "file": { "key": "obj/Debug/net7.0/src.AssemblyInfo.cs" },
          "range": {
            "lineBegin": 19,
            "columnBegin": 12,
            "lineEnd": 19,
            "columnEnd": 17
          }
        }
      }
    }
  },
  {
    "key": {
      "symbol": { "key": "scip-dotnet nuget . . System/" },
      "location": {
        "key": {
          "file": { "key": "obj/Debug/net7.0/src.GlobalUsings.g.cs" },
          "range": {
            "lineBegin": 2,
            "columnBegin": 22,
            "lineEnd": 2,
            "columnEnd": 27
          }
        }
      }
    }
  },
  {
    "key": {
      "symbol": { "key": "scip-dotnet nuget . . System/" },
      "location": {
        "key": {
          "file": { "key": "obj/Debug/net7.0/src.GlobalUsings.g.cs" },
          "range": {
            "lineBegin": 3,
            "columnBegin": 22,
            "lineEnd": 3,
            "columnEnd": 27
          }
        }
      }
    }
  },
  {
    "key": {
      "symbol": { "key": "scip-dotnet nuget . . System/" },
      "location": {
        "key": {
          "file": { "key": "obj/Debug/net7.0/src.GlobalUsings.g.cs" },
          "range": {
            "lineBegin": 4,
            "columnBegin": 22,
            "lineEnd": 4,
            "columnEnd": 27
          }
        }
      }
    }
  },
  {
    "key": {
      "symbol": { "key": "scip-dotnet nuget . . System/" },
      "location": {
        "key": {
          "file": { "key": "obj/Debug/net7.0/src.GlobalUsings.g.cs" },
          "range": {
            "lineBegin": 5,
            "columnBegin": 22,
            "lineEnd": 5,
            "columnEnd": 27
          }
        }
      }
    }
  },
  {
    "key": {
      "symbol": { "key": "scip-dotnet nuget . . System/" },
      "location": {
        "key": {
          "file": { "key": "obj/Debug/net7.0/src.GlobalUsings.g.cs" },
          "range": {
            "lineBegin": 6,
            "columnBegin": 22,
            "lineEnd": 6,
            "columnEnd": 27
          }
        }
      }
    }
  },
  {
    "key": {
      "symbol": { "key": "scip-dotnet nuget . . System/" },
      "location": {
        "key": {
          "file": { "key": "obj/Debug/net7.0/src.GlobalUsings.g.cs" },
          "range": {
            "lineBegin": 7,
            "columnBegin": 22,
            "lineEnd": 7,
            "columnEnd": 27
          }
        }
      }
    }
  },
  {
    "key": {
      "symbol": { "key": "scip-dotnet nuget . . System/" },
      "location": {
        "key": {
          "file": { "key": "obj/Debug/net7.0/src.GlobalUsings.g.cs" },
          "range": {
            "lineBegin": 8,
            "columnBegin": 22,
            "lineEnd": 8,
            "columnEnd": 27
          }
        }
      }
    }
  },
  {
    "key": {
      "symbol": { "key": "scip-dotnet nuget . . Tasks/" },
      "location": {
        "key": {
          "file": { "key": "obj/Debug/net7.0/src.GlobalUsings.g.cs" },
          "range": {
            "lineBegin": 8,
            "columnBegin": 39,
            "lineEnd": 8,
            "columnEnd": 43
          }
        }
      }
    }
  },
  {
    "key": {
      "symbol": { "key": "scip-dotnet nuget . . Threading/" },
      "location": {
        "key": {
          "file": { "key": "obj/Debug/net7.0/src.GlobalUsings.g.cs" },
          "range": {
            "lineBegin": 7,
            "columnBegin": 29,
            "lineEnd": 7,
            "columnEnd": 37
          }
        }
      }
    }
  },
  {
    "key": {
      "symbol": { "key": "scip-dotnet nuget . . Threading/" },
      "location": {
        "key": {
          "file": { "key": "obj/Debug/net7.0/src.GlobalUsings.g.cs" },
          "range": {
            "lineBegin": 8,
            "columnBegin": 29,
            "lineEnd": 8,
            "columnEnd": 37
          }
        }
      }
    }
  },
  {
    "key": {
      "symbol": { "key": "scip-dotnet nuget . . Versioning/" },
      "location": {
        "key": {
          "file": {
            "key": "obj/Debug/net7.0/.NETCoreApp,Version=v7.0.AssemblyAttributes.cs"
          },
          "range": {
            "lineBegin": 4,
            "columnBegin": 35,
            "lineEnd": 4,
            "columnEnd": 44
          }
        }
      }
    }
  },
  {
    "key": {
      "symbol": { "key": "scip-dotnet nuget . . ``/" },
      "location": {
        "key": {
          "file": {
            "key": "obj/Debug/net7.0/.NETCoreApp,Version=v7.0.AssemblyAttributes.cs"
          },
          "range": {
            "lineBegin": 4,
            "columnBegin": 12,
            "lineEnd": 4,
            "columnEnd": 17
          }
        }
      }
    }
  },
  {
    "key": {
      "symbol": { "key": "scip-dotnet nuget . . ``/" },
      "location": {
        "key": {
          "file": { "key": "obj/Debug/net7.0/src.GlobalUsings.g.cs" },
          "range": {
            "lineBegin": 2,
            "columnBegin": 14,
            "lineEnd": 2,
            "columnEnd": 19
          }
        }
      }
    }
  },
  {
    "key": {
      "symbol": { "key": "scip-dotnet nuget . . ``/" },
      "location": {
        "key": {
          "file": { "key": "obj/Debug/net7.0/src.GlobalUsings.g.cs" },
          "range": {
            "lineBegin": 3,
            "columnBegin": 14,
            "lineEnd": 3,
            "columnEnd": 19
          }
        }
      }
    }
  },
  {
    "key": {
      "symbol": { "key": "scip-dotnet nuget . . ``/" },
      "location": {
        "key": {
          "file": { "key": "obj/Debug/net7.0/src.GlobalUsings.g.cs" },
          "range": {
            "lineBegin": 4,
            "columnBegin": 14,
            "lineEnd": 4,
            "columnEnd": 19
          }
        }
      }
    }
  },
  {
    "key": {
      "symbol": { "key": "scip-dotnet nuget . . ``/" },
      "location": {
        "key": {
          "file": { "key": "obj/Debug/net7.0/src.GlobalUsings.g.cs" },
          "range": {
            "lineBegin": 5,
            "columnBegin": 14,
            "lineEnd": 5,
            "columnEnd": 19
          }
        }
      }
    }
  },
  {
    "key": {
      "symbol": { "key": "scip-dotnet nuget . . ``/" },
      "location": {
        "key": {
          "file": { "key": "obj/Debug/net7.0/src.GlobalUsings.g.cs" },
          "range": {
            "lineBegin": 6,
            "columnBegin": 14,
            "lineEnd": 6,
            "columnEnd": 19
          }
        }
      }
    }
  },
  {
    "key": {
      "symbol": { "key": "scip-dotnet nuget . . ``/" },
      "location": {
        "key": {
          "file": { "key": "obj/Debug/net7.0/src.GlobalUsings.g.cs" },
          "range": {
            "lineBegin": 7,
            "columnBegin": 14,
            "lineEnd": 7,
            "columnEnd": 19
          }
        }
      }
    }
  },
  {
    "key": {
      "symbol": { "key": "scip-dotnet nuget . . ``/" },
      "location": {
        "key": {
          "file": { "key": "obj/Debug/net7.0/src.GlobalUsings.g.cs" },
          "range": {
            "lineBegin": 8,
            "columnBegin": 14,
            "lineEnd": 8,
            "columnEnd": 19
          }
        }
      }
    }
  },
  {
    "key": {
      "symbol": { "key": "scip-dotnet nuget . . ``/Dog#" },
      "location": {
        "key": {
          "file": { "key": "Program.cs" },
          "range": {
            "lineBegin": 62,
            "columnBegin": 21,
            "lineEnd": 62,
            "columnEnd": 23
          }
        }
      }
    }
  },
  {
    "key": {
      "symbol": { "key": "scip-dotnet nuget . . ``/Human#" },
      "location": {
        "key": {
          "file": { "key": "Program.cs" },
          "range": {
            "lineBegin": 65,
            "columnBegin": 21,
            "lineEnd": 65,
            "columnEnd": 25
          }
        }
      }
    }
  },
  {
    "key": {
      "symbol": { "key": "scip-dotnet nuget . . ``/Mammal#" },
      "location": {
        "key": {
          "file": { "key": "Program.cs" },
          "range": {
            "lineBegin": 43,
            "columnBegin": 20,
            "lineEnd": 43,
            "columnEnd": 25
          }
        }
      }
    }
  },
  {
    "key": {
      "symbol": { "key": "scip-dotnet nuget . . ``/Mammal#" },
      "location": {
        "key": {
          "file": { "key": "Program.cs" },
          "range": {
            "lineBegin": 50,
            "columnBegin": 22,
            "lineEnd": 50,
            "columnEnd": 27
          }
        }
      }
    }
  },
  {
    "key": {
      "symbol": { "key": "scip-dotnet nuget . . ``/Mammal#" },
      "location": {
        "key": {
          "file": { "key": "Program.cs" },
          "range": {
            "lineBegin": 59,
            "columnBegin": 21,
            "lineEnd": 59,
            "columnEnd": 26
          }
        }
      }
    }
  },
  {
    "key": {
      "symbol": { "key": "scip-dotnet nuget . . ``/Mammal#NumberOfLegs." },
      "location": {
        "key": {
          "file": { "key": "Program.cs" },
          "range": {
            "lineBegin": 60,
            "columnBegin": 41,
            "lineEnd": 60,
            "columnEnd": 52
          }
        }
      }
    }
  },
  {
    "key": {
      "symbol": { "key": "scip-dotnet nuget . . ``/Mammal#NumberOfLegs." },
      "location": {
        "key": {
          "file": { "key": "Program.cs" },
          "range": {
            "lineBegin": 63,
            "columnBegin": 47,
            "lineEnd": 63,
            "columnEnd": 58
          }
        }
      }
    }
  },
  {
    "key": {
      "symbol": { "key": "scip-dotnet nuget . . ``/Mammal#NumberOfLegs." },
      "location": {
        "key": {
          "file": { "key": "Program.cs" },
          "range": {
            "lineBegin": 66,
            "columnBegin": 47,
            "lineEnd": 66,
            "columnEnd": 58
          }
        }
      }
    }
  },
  {
    "key": {
      "symbol": { "key": "scip-dotnet nuget . . ``/Mammal#Specie." },
      "location": {
        "key": {
          "file": { "key": "Program.cs" },
          "range": {
            "lineBegin": 60,
            "columnBegin": 26,
            "lineEnd": 60,
            "columnEnd": 31
          }
        }
      }
    }
  },
  {
    "key": {
      "symbol": { "key": "scip-dotnet nuget . . ``/Mammal#Specie." },
      "location": {
        "key": {
          "file": { "key": "Program.cs" },
          "range": {
            "lineBegin": 63,
            "columnBegin": 32,
            "lineEnd": 63,
            "columnEnd": 37
          }
        }
      }
    }
  },
  {
    "key": {
      "symbol": { "key": "scip-dotnet nuget . . ``/Mammal#Specie." },
      "location": {
        "key": {
          "file": { "key": "Program.cs" },
          "range": {
            "lineBegin": 66,
            "columnBegin": 32,
            "lineEnd": 66,
            "columnEnd": 37
          }
        }
      }
    }
  },
  {
    "key": {
      "symbol": {
        "key": "scip-dotnet nuget . . ``/Mammal#`.ctor`(+1).(numberOfLegs)"
      },
      "location": {
        "key": {
          "file": { "key": "Program.cs" },
          "range": {
            "lineBegin": 22,
            "columnBegin": 23,
            "lineEnd": 22,
            "columnEnd": 34
          }
        }
      }
    }
  },
  {
    "key": {
      "symbol": {
        "key": "scip-dotnet nuget . . ``/Mammal#`.ctor`(+1).(specie)"
      },
      "location": {
        "key": {
          "file": { "key": "Program.cs" },
          "range": {
            "lineBegin": 23,
            "columnBegin": 17,
            "lineEnd": 23,
            "columnEnd": 22
          }
        }
      }
    }
  },
  {
    "key": {
      "symbol": { "key": "scip-dotnet nuget . . ``/Mammal#numberOfLegs." },
      "location": {
        "key": {
          "file": { "key": "Program.cs" },
          "range": {
            "lineBegin": 22,
            "columnBegin": 8,
            "lineEnd": 22,
            "columnEnd": 19
          }
        }
      }
    }
  },
  {
    "key": {
      "symbol": { "key": "scip-dotnet nuget . . ``/Mammal#numberOfLegs." },
      "location": {
        "key": {
          "file": { "key": "Program.cs" },
          "range": {
            "lineBegin": 30,
            "columnBegin": 11,
            "lineEnd": 30,
            "columnEnd": 22
          }
        }
      }
    }
  },
  {
    "key": {
      "symbol": { "key": "scip-dotnet nuget . . ``/Mammal#specie." },
      "location": {
        "key": {
          "file": { "key": "Program.cs" },
          "range": {
            "lineBegin": 17,
            "columnBegin": 3,
            "lineEnd": 17,
            "columnEnd": 8
          }
        }
      }
    }
  },
  {
    "key": {
      "symbol": { "key": "scip-dotnet nuget . . ``/Mammal#specie." },
      "location": {
        "key": {
          "file": { "key": "Program.cs" },
          "range": {
            "lineBegin": 23,
            "columnBegin": 8,
            "lineEnd": 23,
            "columnEnd": 13
          }
        }
      }
    }
  },
  {
    "key": {
      "symbol": { "key": "scip-dotnet nuget . . ``/Mammal#specie." },
      "location": {
        "key": {
          "file": { "key": "Program.cs" },
          "range": {
            "lineBegin": 38,
            "columnBegin": 11,
            "lineEnd": 38,
            "columnEnd": 16
          }
        }
      }
    }
  },
  {
    "key": {
      "symbol": {
        "key": "scip-dotnet nuget System.Console 7.0.0.0 System/Console#"
      },
      "location": {
        "key": {
          "file": { "key": "Program.cs" },
          "range": {
            "lineBegin": 60,
            "columnBegin": 3,
            "lineEnd": 60,
            "columnEnd": 9
          }
        }
      }
    }
  },
  {
    "key": {
      "symbol": {
        "key": "scip-dotnet nuget System.Console 7.0.0.0 System/Console#"
      },
      "location": {
        "key": {
          "file": { "key": "Program.cs" },
          "range": {
            "lineBegin": 63,
            "columnBegin": 9,
            "lineEnd": 63,
            "columnEnd": 15
          }
        }
      }
    }
  },
  {
    "key": {
      "symbol": {
        "key": "scip-dotnet nuget System.Console 7.0.0.0 System/Console#"
      },
      "location": {
        "key": {
          "file": { "key": "Program.cs" },
          "range": {
            "lineBegin": 66,
            "columnBegin": 9,
            "lineEnd": 66,
            "columnEnd": 15
          }
        }
      }
    }
  },
  {
    "key": {
      "symbol": {
        "key": "scip-dotnet nuget System.Console 7.0.0.0 System/Console#WriteLine(+11)."
      },
      "location": {
        "key": {
          "file": { "key": "Program.cs" },
          "range": {
            "lineBegin": 60,
            "columnBegin": 11,
            "lineEnd": 60,
            "columnEnd": 19
          }
        }
      }
    }
  },
  {
    "key": {
      "symbol": {
        "key": "scip-dotnet nuget System.Console 7.0.0.0 System/Console#WriteLine(+11)."
      },
      "location": {
        "key": {
          "file": { "key": "Program.cs" },
          "range": {
            "lineBegin": 63,
            "columnBegin": 17,
            "lineEnd": 63,
            "columnEnd": 25
          }
        }
      }
    }
  },
  {
    "key": {
      "symbol": {
        "key": "scip-dotnet nuget System.Console 7.0.0.0 System/Console#WriteLine(+11)."
      },
      "location": {
        "key": {
          "file": { "key": "Program.cs" },
          "range": {
            "lineBegin": 66,
            "columnBegin": 17,
            "lineEnd": 66,
            "columnEnd": 25
          }
        }
      }
    }
  },
  {
    "key": {
      "symbol": {
        "key": "scip-dotnet nuget System.Runtime 7.0.0.0 Reflection/AssemblyCompanyAttribute#`.ctor`()."
      },
      "location": {
        "key": {
          "file": { "key": "obj/Debug/net7.0/src.AssemblyInfo.cs" },
          "range": {
            "lineBegin": 13,
            "columnBegin": 30,
            "lineEnd": 13,
            "columnEnd": 53
          }
        }
      }
    }
  },
  {
    "key": {
      "symbol": {
        "key": "scip-dotnet nuget System.Runtime 7.0.0.0 Reflection/AssemblyConfigurationAttribute#`.ctor`()."
      },
      "location": {
        "key": {
          "file": { "key": "obj/Debug/net7.0/src.AssemblyInfo.cs" },
          "range": {
            "lineBegin": 14,
            "columnBegin": 30,
            "lineEnd": 14,
            "columnEnd": 59
          }
        }
      }
    }
  },
  {
    "key": {
      "symbol": {
        "key": "scip-dotnet nuget System.Runtime 7.0.0.0 Reflection/AssemblyFileVersionAttribute#`.ctor`()."
      },
      "location": {
        "key": {
          "file": { "key": "obj/Debug/net7.0/src.AssemblyInfo.cs" },
          "range": {
            "lineBegin": 15,
            "columnBegin": 30,
            "lineEnd": 15,
            "columnEnd": 57
          }
        }
      }
    }
  },
  {
    "key": {
      "symbol": {
        "key": "scip-dotnet nuget System.Runtime 7.0.0.0 Reflection/AssemblyInformationalVersionAttribute#`.ctor`()."
      },
      "location": {
        "key": {
          "file": { "key": "obj/Debug/net7.0/src.AssemblyInfo.cs" },
          "range": {
            "lineBegin": 16,
            "columnBegin": 30,
            "lineEnd": 16,
            "columnEnd": 66
          }
        }
      }
    }
  },
  {
    "key": {
      "symbol": {
        "key": "scip-dotnet nuget System.Runtime 7.0.0.0 Reflection/AssemblyProductAttribute#`.ctor`()."
      },
      "location": {
        "key": {
          "file": { "key": "obj/Debug/net7.0/src.AssemblyInfo.cs" },
          "range": {
            "lineBegin": 17,
            "columnBegin": 30,
            "lineEnd": 17,
            "columnEnd": 53
          }
        }
      }
    }
  },
  {
    "key": {
      "symbol": {
        "key": "scip-dotnet nuget System.Runtime 7.0.0.0 Reflection/AssemblyTitleAttribute#`.ctor`()."
      },
      "location": {
        "key": {
          "file": { "key": "obj/Debug/net7.0/src.AssemblyInfo.cs" },
          "range": {
            "lineBegin": 18,
            "columnBegin": 30,
            "lineEnd": 18,
            "columnEnd": 51
          }
        }
      }
    }
  },
  {
    "key": {
      "symbol": {
        "key": "scip-dotnet nuget System.Runtime 7.0.0.0 Reflection/AssemblyVersionAttribute#`.ctor`()."
      },
      "location": {
        "key": {
          "file": { "key": "obj/Debug/net7.0/src.AssemblyInfo.cs" },
          "range": {
            "lineBegin": 19,
            "columnBegin": 30,
            "lineEnd": 19,
            "columnEnd": 53
          }
        }
      }
    }
  },
  {
    "key": {
      "symbol": {
        "key": "scip-dotnet nuget System.Runtime 7.0.0.0 System/String#Empty."
      },
      "location": {
        "key": {
          "file": { "key": "Program.cs" },
          "range": {
            "lineBegin": 14,
            "columnBegin": 33,
            "lineEnd": 14,
            "columnEnd": 37
          }
        }
      }
    }
  },
  {
    "key": {
      "symbol": {
        "key": "scip-dotnet nuget System.Runtime 7.0.0.0 Versioning/TargetFrameworkAttribute#FrameworkDisplayName."
      },
      "location": {
        "key": {
          "file": {
            "key": "obj/Debug/net7.0/.NETCoreApp,Version=v7.0.AssemblyAttributes.cs"
          },
          "range": {
            "lineBegin": 4,
            "columnBegin": 99,
            "lineEnd": 4,
            "columnEnd": 118
          }
        }
      }
    }
  },
  {
    "key": {
      "symbol": {
        "key": "scip-dotnet nuget System.Runtime 7.0.0.0 Versioning/TargetFrameworkAttribute#`.ctor`()."
      },
      "location": {
        "key": {
          "file": {
            "key": "obj/Debug/net7.0/.NETCoreApp,Version=v7.0.AssemblyAttributes.cs"
          },
          "range": {
            "lineBegin": 4,
            "columnBegin": 46,
            "lineEnd": 4,
            "columnEnd": 69
          }
        }
      }
    }
  }
]